sdk/digitaltwins/Azure.DigitalTwins.Core/samples/Readme.md

```diff
@@ -30,37 +30,22 @@ var dtClient = new DigitalTwinsClient(
     tokenCredential);
 ```
 
-Also, if you need to override pipeline behavior, such as provide your own HttpClient instance, you can do that via client options. This parameter is optional.
+Also, if you need to override pipeline behavior, such as provide your own HttpClient instance, you can do that via the other constructor that takes a client options.
+It provides an opportunity to override default behavior including:
 
-```C# Snippet:DigitalTwinsSampleCreateServiceClientInteractiveLogin
```

Comment:
Azure SDK team said leave out example for client options unless it is for ADT-specific options, and simply refer to Azure.Core documentation for client options base support.